gateway/ln-gateway/src/lib.rs

```diff
@@ -188,6 +188,14 @@ impl LnGateway {
         Ok(actor)
     }
 
+    fn select_actor(&self, federation_id: FederationId) -> Result<Arc<GatewayActor>> {
+        // TODO: Select federation client based on federation id
+        self.actors
+            .get(&federation_id)
+            .cloned()
+            .ok_or(LnGatewayError::UnknownFederation)
+    }
```

Comment:
Isn't it doing exactly this here?

Reply:
Oh, I missed cleaning up the TODO comment!